MANO-API/src/main/java/com/ubiqube/etsi/mano/service/rest/NfvoRest.java

```diff
@@ -5,15 +5,18 @@
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.stereotype.Service;
 import org.springframework.util.Assert;
+import org.springframework.beans.factory.annotation.Value;
+import javax.annotation.PostConstruct;
 
-import com.ubiqube.etsi.mano.service.Configuration;
 
 @Service
 public class NfvoRest extends AbstractRest {
-	private final String url;
 
-	public NfvoRest(final Configuration _conf) {
-		url = _conf.get("nfvo.url");
+	@Value("${nfvo.url}")
```

Comment:
Configuration is te way to go, it's have externalized configuration.
Note that we don't really know how we handle configurations for the moment we are pleased with simple property files. But we probably quickly want configuration hot swap, and so on...